instructions.md

```diff
@@ -50,13 +51,12 @@ The first step will be uploading the images into a docker image repository onto
 
 - Open your docker terminal if you are on Windows. I believe Mac's have the docker machine initialized automatically so you can go to a normal terminal.
 Run the aws cli command listed on the push instructions:
```

Comment:
You may still want to leave the old `aws ecr-login` command

Reply:
Didn't know I deleted it.

Reply:
Thats why we review. I've totally done that before. 👍

Reply:
fixed it.  thanks for catching that